tools/elfutils/Makefile

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Please use SPDX conform header.

Reply:
Right, will change both this and for `tools/dwarves/Makefile`. 

BTW, is SPDX a recent policy thing? I noticed most  utilities in `tools/` aren't using it and followed their example.

Reply:
It's a recent thing and I'll just try to enforce it wherever something new is added, it's not a official policy.